src/inmanta/agent/io/local.py

```diff
@@ -22,7 +22,13 @@
 import subprocess
 import sys
 
-from inmanta.stable_api import stable_api
```

Comment:
I would remove the decorator completely and just add a comment, this code has to be python2 compatible as well I think.